webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -1,55 +1,53 @@
 <template>
   <v-app id="web-conferencing-admin">
-    <v-container style="width: 95%" class=" white card-border-radius pa-5">
+    <v-card class="px-6 card-border-radius overflow-hidden" flat>
       <div
         v-show="error"
         class="alert alert-error">
         {{ $t(`${errorResourceBase}.${error}`) ? $t(`${errorResourceBase}.${error}`) : error }}
       </div>
-      <v-row>
-        <v-col>
-          <v-label>
-            <span class="text-color font-weight-bold mb-2">{{ $t("webconferencing.admin.title") }}</span>
-          </v-label>
-        </v-col>
-      </v-row>
-      <v-row class="mx-0 mt-2">
-        <v-col class="d-flex flex-row px-0 py-0 col-10">
-          <span class="text-color"> {{ $t("webconferencing.admin.section.title") }} </span>
-        </v-col>
-        <v-list>
-          <v-list-item
-            class="px-0"
-            v-for="providerConfig in providersConfig"
-            :key="providerConfig.title">
-            <v-list-item-action class="me-2 mt-0">
-              <v-switch
-                :dense="true"
-                :input-value="providerConfig.active"
-                :ripple="true"
-                v-model="providerConfig.active"
-                color="primary"
-                inset
-                class="providersSwitcher"
-                @change="changeActive(providerConfig)" />
-            </v-list-item-action>
-            <v-list-item-content class="py-0 pt-1">
-              <v-list-item-title>
-                {{ $t(`webconferencing.admin.${providerConfig.title}.name`)
-                  ? $t(`webconferencing.admin.${providerConfig.title}.name`)
-                  : providerConfig.title }} 
-              </v-list-item-title>
-              <div
-                class="caption text-light-color" 
-                v-html="$t(`webconferencing.admin.${providerConfig.title}.description`)
-                  ? $t(`webconferencing.admin.${providerConfig.title}.description`)
-                  : '' ">
-              </div>
-            </v-list-item-content>
-          </v-list-item>
-        </v-list>
-      </v-row>
-    </v-container>
+      <v-list-item class="px-0 mb-2 mt-4">
+        <v-list-item-content class="py-0">
+          <v-list-item-title class="my-0">
+            <h4 class="font-weight-bold mt-0">
+              {{ $t("webconferencing.admin.title") }}
+            </h4>
+          </v-list-item-title>
+          <v-list-item-subtitle class="pt-4">
+            <h4 class="my-0 text-color">{{ $t("webconferencing.admin.section.title") }}</h4>
+          </v-list-item-subtitle>
+        </v-list-item-content>
+      </v-list-item>
+      <div class="pb-4">
```

Comment:
Why do you keep a div here ? Is there a particular reason ?
In addition, you add padding on the div an on the v-list item.

Reply:
> Why do you keep a div here ? Is there a particular reason ? In addition, you add padding on the div an on the v-list item.

In order to add bottom padding for the whole list  and also apply bottom padding to each individual item in the list.

Reply:
Can we only keep bottom padding for each element ? it will also add a bottom padding for the last one no ?

Reply:
> Can we only keep bottom padding for each element ? it will also add a bottom padding for the last one no ?

Done!